<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-09-25T11:43:21 PDT" name="Suite" finished-at="2023-09-25T11:43:38 PDT" duration-ms="17475">
    <groups>
      <group name="errorHandling">
        <method signature="NegativeScenario.negativeTest()[pri:0, instance:e2eTesting.NegativeScenario@49964d75]" name="negativeTest" class="e2eTesting.NegativeScenario"/>
      </group> <!-- errorHandling -->
    </groups>
    <test started-at="2023-09-25T11:43:21 PDT" name="Test" finished-at="2023-09-25T11:43:38 PDT" duration-ms="17475">
      <class name="e2eTesting.NegativeScenario">
        <test-method is-config="true" signature="launchURL()[pri:0, instance:e2eTesting.NegativeScenario@49964d75]" started-at="2023-09-25T11:43:21 PDT" name="launchURL" finished-at="2023-09-25T11:43:36 PDT" duration-ms="14926" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchURL -->
        <test-method signature="negativeTest()[pri:0, instance:e2eTesting.NegativeScenario@49964d75]" started-at="2023-09-25T11:43:36 PDT" name="negativeTest" finished-at="2023-09-25T11:43:37 PDT" duration-ms="1520" status="FAIL">
          <exception class="java.lang.Error">
            <message>
              <![CDATA[Unresolved compilation problems: 
Lambda expressions are allowed only at source level 1.8 or above
Cannot refer to the non-final local variable product defined in an enclosing scope
]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Error: Unresolved compilation problems: 
Lambda expressions are allowed only at source level 1.8 or above
Cannot refer to the non-final local variable product defined in an enclosing scope

at e2eTesting.PageObjects.ProductCatalogue.<init>(ProductCatalogue.java:41)
at e2eTesting.PageObjects.LandingPage.LoginToApplication(LandingPage.java:39)
at e2eTesting.NegativeScenario.negativeTest(NegativeScenario.java:13)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Error -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- negativeTest -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:e2eTesting.NegativeScenario@49964d75]" started-at="2023-09-25T11:43:38 PDT" name="tearDown" finished-at="2023-09-25T11:43:38 PDT" duration-ms="171" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- e2eTesting.NegativeScenario -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
